src/main/java/net/minecraftforge/client/gui/ScrollPanel.java

```diff
@@ -48,12 +48,12 @@
     private boolean scrolling;
     protected float scrollDistance;
     protected boolean captureMouse = true;
-    protected final int border = 4;
+    protected final int border;
 
     private final int barWidth = 6;
     private final int barLeft;
 
-    public ScrollPanel(Minecraft client, int width, int height, int top, int left)
+    public ScrollPanel(Minecraft client, int width, int height, int top, int left, int border)
```

Comment:
This is a breaking change, as any mod which links to the constructor without the new parameter will fail. Create an overload of this constructor without the new `border` parmaeter which passes in a default value for it (such as the previously present value, `4`).